src/components/ArtistMediumCardCollection.jsx

```diff
@@ -0,0 +1,17 @@
+import ArtistsMediumCard from "./ArtistsMediumCard";
+import "./ArtistMediumCardCollection.css";
+import PropTypes from "prop-types";
+
+export default function ArtistMediumCardCollection(artists) {
+  return (
+    <div className="artists-collection">
+      {artists.followedArtists.map((artist) => (
```

Comment:
This is a little misleading. You have a proptype defined called artist, but really you're using artists.followedArtists`.. You might want to dereference that one level up so that artists is alll you would really need.